app/i18n/locales/pt-BR.js

```diff
@@ -440,5 +440,8 @@ export default {
 	Server_selection: 'Seleção de servidor',
 	Server_selection_numbers: 'Selecionar servidor 1...9',
 	Add_server: 'Adicionar servidor',
-	New_line: 'Nova linha'
+	New_line: 'Nova linha',
```

Comment:
I couldn't find this key anywhere else.
Can we remove it?

Reply:
Sure, I've deleted this Intl for `de, en, nl, pt-Br`

Reply:
Nevermind.
It's my bad.